openfe/tests/protocols/test_openmm_plain_md_protocols.py

```diff
@@ -177,8 +177,29 @@ def test_dry_run_gaff_vacuum(benzene_vacuum_system, tmpdir):
     )
     unit = list(dag.protocol_units)[0]
 
-    with tmpdir.as_cwd():
-        system = unit.run(dry=True)['debug']['system']
+    # If we do a lot of GAFF testing, this should be refactored so we don't
```

Comment:
As above on an issue number in the comment please 🙏🏽

Reply:
Of course

Reply:
done with https://github.com/OpenFreeEnergy/openfe/pull/847/commits/badb1020783268e8532f9413e18e23223fbe4bd0